internal/cmd/build/prompt.go

```diff
@@ -9,18 +9,162 @@ import (
 )
 
 // prompt invokes bedrock to produce a template based on the prompt
-func prompt(p string) {
-	prompt := fmt.Sprintf("Write an AWS CloudFormation YAML template that builds the following:\n\n%s\n\nDo not include any explanation.\n\nWrite only the content of the YAML file.\n\nOutput valid YAML within <yaml></yaml> tags.", p)
-	config.Debugf("About to invoke bedrock claude2 with prompt: %s", prompt)
-	r, err := bedrock.Invoke(prompt)
-	if err != nil {
-		panic(err)
+func runPrompt(p string) {
+	mid := modelId(model)
+	if guard {
```

Comment:
just a little thing, instead of if a switch case here would keep it clear even if there are more options in the future

Reply:
Yeah, maybe having guard and rego as bools is not the most scalable.. Maybe something like --filetype "guard|rego|yaml" ?

Reply:
yea that looks good, and the default value could be cfn :+1:

Reply:
Done